engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/stores/ExecutionStore.scala

```diff
@@ -274,24 +281,24 @@ sealed abstract class ExecutionStore private[stores](statusStore: Map[JobKey, Ex
         // Even if the key is runnable, if it's a call key and there's already too many queued jobs,
         // don't start it and mark it as WaitingForQueueSpace
         // TODO maybe also limit the number of expression keys to run somehow ?
-        case callKey: CallKey if runnable && queuedJobsAboveThreshold =>
+        case callKey: CallKey if runnable && startableJobLimit > 0 =>
```

Comment:
I think `startableJobLimit > 0` leaves us room for starting more jobs right away? So it should be `if runnable && startableJobLimit <= 0` here?

Reply:
With the test case passing, I think this is the right way round, but maybe the variable is badly named? It's supposed to mean "how many jobs am I going to be allowing myself to start this time". Which you need to be over 0 to start any new jobs

Reply:
> It's supposed to mean "how many jobs am I going to be allowing myself to start this time". Which you need to be over 0 to start any new jobs

Exactly, and this is how I read it. But I think in this case clause we are saying that the job is not runnable this time (by returning false) and postponing the job by transitioning it into the `WaitingForQueueSpace` state.

Reply:
Oh right, I see what you mean. Yes I think you're right, I'll fix that